tests/InjectorTest.php

```diff
@@ -408,4 +433,115 @@ public function testUnnamedScalarParam(): void
 
         (new Injector($container))->invoke($getEngineName, ['test']);
     }
+
+    /**
+     * Constructor method not defined
+     */
+    public function testMakeWithoutConstructor(): void
+    {
+        $container = new Container([]);
+
+        $object = (new Injector($container))->make(MakeNoConstructor::class);
+
+        $this->assertInstanceOf(MakeNoConstructor::class, $object);
+    }
+
+    /**
+     * Constructor without arguments
+     */
+    public function testMakeWithoutArguments(): void
+    {
+        $container = new Container([]);
+
+        $object = (new Injector($container))->make(MakeEmptyConstructor::class);
+
+        $this->assertInstanceOf(MakeEmptyConstructor::class, $object);
+    }
+
+    /**
+     * Private constructor unavailable from Injector context
```

Comment:
I think it could be helpful

Reply:
You can create an issue for discussion about this